modules/subcluster/outputs.tf

```diff
@@ -2,7 +2,6 @@ output "applications" {
   value = compact([
     juju_application.ams.id,
     juju_application.lxd.id,
-    juju_application.ams_node_controller.id,
```

Comment:
We brought it back following the recent discussion, as the node controller can be used for RT scheduling.
Maybe double check with team on this as moving forwrad